src/sage/schemes/elliptic_curves/ell_finite_field.py

```diff
@@ -1765,6 +1777,14 @@ def twists(self):
              (0, 0, z8^6 + z8^3 + z8^2, 0, 0),
              (0, 0, z8^6 + z8^3 + z8^2, 0, z8^3 + z8^2)]
 
+        The order of the quadratic twist is precomputed, so the second
+        computation below is fast::
+
+            sage: E = EllipticCurve(GF(2^255 - 19), [3, 5])
+            sage: E.set_order(2^255 - 94400682670973290858718002131957303262)
+            sage: E.quadratic_twist().order()
```

Comment:
Maybe instead of this you can show `E.quadratic_twist()._order` works (or with a hasattr to show the order is already there?